platform/ethereum/api_test.go

```diff
@@ -431,8 +431,8 @@ var collection2Dst = blockatlas.Collection{
 	Description:     "Year 4424: The search begins for new life on the other side of the galaxy. Explore abandoned space stations, mysterious caverns, and ruins on far away worlds in order to unlock puzzles and secrets! Beware the rogue machines!",
 	ExternalLink:    "https://www.ageofrust.games/",
 	Total:           1,
-	CategoryAddress: "0xfaafdc07907ff5120a76b34b731b278c38d6043c---age-of-rust",
-	Id:              "0xfaafdc07907ff5120a76b34b731b278c38d6043c---age-of-rust",
+	CategoryAddress: "0xfaafdc07907ff5120a76b34b731b278c38d6043c",
```

Comment:
`ID` and `CategoryAddress` should be the same value and `CategoryAddress ` will be removed in the future